test/testdata/test-program.yaml

```diff
@@ -12,7 +12,7 @@ resources:
                 annotations:
                     pulumi.com/patchForce: "true"
                 name:
-                    test-configmap
+                    test-configmap-valid-test
```

Comment:
Prevent deletion race condition, as another spec was previously creating a CM with the same name as well.